Classes/Base/Context.php

```diff
@@ -72,7 +72,12 @@ protected static function addCache( \Aimeos\MShop\Context\Item\Iface $context )
 			return $fcn( $context );
 		}
 
-		switch( \Aimeos\Aimeos\Base::getExtConfig( 'cacheName', 'Typo3' ) )
+		$cacheName = \Aimeos\Aimeos\Base::getExtConfig( 'cacheName', 'Typo3' );
+		if ( $GLOBALS['TSFE'] && $GLOBALS['TSFE']->headerNoCache() ) {
```

Comment:
Can you please use `isset( $GLOBALS['TSFE'] )`? This code can lead to a warning about accessing a non-existing property

Reply:
You are completely right. I did not think of that.